truffle.js

```diff
@@ -3,11 +3,13 @@ require('ts-node/register/transpile-only')
 var HDWalletProvider = require('truffle-hdwallet-provider')
 var mnemonic = 'digital unknown jealous mother legal hedgehog save glory december universe spread figure custom found six'
 
-const secretMnemonicFile = './secret_mnemonic'
+// NOGIT
```

Comment:
I don't think it worked...

Reply:
the marker was !NOGIT... I relaxed it... removing this file